app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,46 @@
+class TeamsController < ApplicationController
+  authorize_persona class_name: "User"
+  before_action :validate_already_existing_user, only: [:create]
+  before_action :same_user, only: [:create, :new, :index]
+  grant(
+    manager: :all
+  )
+
+  def index
+    @team = current_user.team_members
+  end
+
+  def new
+  end
+
+  def create
+    if current_user.max_team_members_reached?
+      flash[:alert] = 'You are not allow to add more members to your team!'
+    else
+      User.invite! email: params[:email],
+                  first_name: params[:first_name],
+                  last_name: params[:last_name],
+                  authorization_tier: :user,
+                  manager: current_user
+      flash[:notice] = "The invitation was send to #{params[:email]}"
```

Comment:
may we use `params.permit` or `params.require`, etc. here?

Reply:
Understood. 

Since you are using DeviseInvitable, that is permitted. 

But try checking out the following thread 

https://github.com/scambra/devise_invitable/issues/690 

You could also modify the DeviseInvitable controller for your needs like additional parameters and permissions.